github/GitTreeElement.py

```diff
@@ -42,6 +43,13 @@ class GitTreeElement(github.GithubObject.NonCompletableGithubObject):
     def __repr__(self):
         return self.get__repr__({"sha": self._sha.value, "path": self._path.value})
 
+    @property
+    def last_modified_at(self):
+        """
+        :type: datetime.datetime
+        """
+        return self._makeDatetimeAttribute(self.last_modified).value
```

Comment:
I think this is the wrong place. Since last_modified comes from GithubObject, which means pretty much everything may have it set, why not add it there?

Reply:
ah of course, missed that.